.github/workflows/ci.yml

```
@@ -328,6 +328,13 @@ jobs:
     # (swift, rust) are pre-installed on the runner host.
     runs-on: [self-hosted, macOS, X64]
     timeout-minutes: 45
+    env:
+      # Isolate the SwiftPM shared cache (--cache-path) per runner instance so
+      # two macOS runners on one host don't race on the shared
+      # ~/Library/Caches/org.swift.swiftpm — the prove-swift flake. RUNNER_TEMP
+      # is per-runner, so concurrent swift builds get distinct caches; .build
+      # (compiled artifacts) is still restored from the GH cache below.
+      SWIFT_CACHE_PATH: ${{ runner.temp }}/swiftpm-cache
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)